Backend/dev/migrations-read-only/rider-app/beckn_config.sql

```diff
@@ -44,4 +44,9 @@ ALTER TABLE atlas_app.beckn_config ADD COLUMN provider text  default 'ONDC';
 
 ------- SQL updates -------
 
-ALTER TABLE atlas_app.beckn_config ADD COLUMN verified_by text  default 'BPP';
\ No newline at end of file
+ALTER TABLE atlas_app.beckn_config ADD COLUMN verified_by text  default 'BPP';
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_app.beckn_config ADD COLUMN verification_cipher text ;
```

Comment:
move this to `IntegratedBPPConfig` table

Reply:
Done